scarb/src/ops/resolve.rs

```diff
@@ -439,10 +445,11 @@ fn check_cairo_version_compatibility(packages: &[Package], ws: &Workspace<'_>) -
 
 fn generate_cairo_plugin_compilation_units(
     member: &Package,
-    ws: &Workspace<'_>,
+    _ws: &Workspace<'_>,
```

Comment:
Did this parameter stay in order to be consistent between `generate_cairo_compilation_units` and this function?

Reply:
Indeed, can be removed.